Resources/config/services.yml

```diff
@@ -40,7 +40,7 @@ services:
             - { name: netdudes_data_sourcery.extension }
 
     netdudes_data_sourcery.extension_container:
-        class: Netdudes\DataSourceryBundle\Extension\TableBundleExtensionContainer
+        class: Netdudes\DataSourceryBundle\Extension\UqlExtensionContainer
```

Comment:
Should this class be called just `ExtensionContainer`? Or maybe we should call the service something like `netdudes_data_sourcery.uql_extension_container` or similar? What do you think?